wagtailsurveys/tests/settings.py

```diff
@@ -53,37 +54,59 @@
     }
 ]
 
-MIDDLEWARE_CLASSES = (
+MIDDLEWARE_CLASSES = [
     'django.middleware.common.CommonMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+]
 
-    'wagtail.wagtailcore.middleware.SiteMiddleware',
-    'wagtail.wagtailredirects.middleware.RedirectMiddleware',
-)
+if WAGTAIL_VERSION < (2, 0):
+    MIDDLEWARE_CLASSES = [
+        'wagtail.wagtailcore.middleware.SiteMiddleware',
```

Comment:
Not sure if it’s critical or not, but previously Wagtail middlewares were at the end of the list.

Reply:
Thanks for spotting that. I've corrected it now